docker/localdashboard.yaml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+    parse-dashboard:
+      image: parseplatform/parse-dashboard:4.2.0-alpha.4
```

Comment:
Same, I would juste your :latest here

Reply:
the last stable version of parse dashboard has login but, but fixed in the alpha version

https://github.com/parse-community/parse-dashboard/issues/2194